src/components/Modal/index.jsx

```diff
@@ -0,0 +1,22 @@
+import styles from "./styles.module.css";
+import { ImCross } from "react-icons/im";
+
+export const Modal = ({ modalRef, openModal, handleModalClose, children }) => {
+  if (!openModal) {
+    return <></>;
```

Comment:
P3:
태그안에 값이 없으면 저는 아래와 같은 방식을 선호합니다.
```suggestion
    return </>;
```